components/patient-data/impl/src/main/java/org/phenotips/data/internal/PhenoTipsGene.java

```diff
@@ -117,7 +117,7 @@
      * @param id gene Ensembl ID
      * @param name gene HGNC vocabulary symbol
      * @param status gene status, one of possible values: "candidate" (default value), "rejected_candidate", "rejected",
-     *            "solved"
+     *            "solved", "candidate_novel_disease", "candidate_novel_phen", "umc", "umc_vus", "umc_msv"
```

Comment:
Comments should not list the available statuses, because they may change. Instead, a link to where the statuses are defined should be provided.